testsuite/MDAnalysisTests/topology/test_pdb.py

```diff
@@ -244,3 +247,104 @@ def test_PDB_metals():
     assert u.atoms[1].mass == pytest.approx(tables.masses["FE"])
     assert u.atoms[2].mass == pytest.approx(tables.masses["CA"])
     assert u.atoms[3].mass == pytest.approx(tables.masses["MG"])
+
+
+PDB_elements = """\
+REMARK For testing reading of elements
+REMARK This model represents an ideal PDB file with valid elements
+ATOM      1  N   ASN A   1      -8.901   4.127  -0.555  1.00  0.00           N
+ATOM      2  CA  ASN A   1      -8.608   3.135  -1.618  1.00  0.00           C
+ATOM      3  C   ASN A   1      -7.117   2.964  -1.897  1.00  0.00           C
+ATOM      4  O   ASN A   1      -6.634   1.849  -1.758  1.00  0.00           O
+ATOM      5  CB  ASN A   1      -9.437   3.396  -2.889  1.00  0.00           C
+ATOM      6  CG  ASN A   1     -10.915   3.130  -2.611  1.00  0.00           C
+ATOM      7  OD1 ASN A   1     -11.269   2.700  -1.524  1.00  0.00           O
+ATOM      8  ND2 ASN A   1     -11.806   3.406  -3.543  1.00  0.00           N
+ATOM      9  H1  ASN A   1      -8.330   3.957   0.261  1.00  0.00           H
+ATOM     10  H2  ASN A   1      -8.740   5.068  -0.889  1.00  0.00           H
+ATOM     11  H3  ASN A   1      -9.877   4.041  -0.293  1.00  0.00           H
+ATOM     12  HA  ASN A   1      -8.930   2.162  -1.239  1.00  0.00           H
+ATOM     13  HB2 ASN A   1      -9.310   4.417  -3.193  1.00  0.00           H
+ATOM     14  HB3 ASN A   1      -9.108   2.719  -3.679  1.00  0.00           H
+ATOM     15 HD21 ASN A   1     -11.572   3.791  -4.444  1.00  0.00           H
+ATOM     16 HD22 ASN A   1     -12.757   3.183  -3.294  1.00  0.00           H
+TER      17
+HETATM   18 CU    CU A   2      03.000  00.000  00.000  1.00 00.00          CU
+HETATM   19 FE    FE A   3      00.000  03.000  00.000  1.00 00.00          Fe
+HETATM   20 Mg    Mg A   4      03.000  03.000  03.000  1.00 00.00          Mg
+HETATM   21 Ca    Ca A   5      00.000  00.000  03.000  1.00 00.00          CA
+TER      22
+HETATM 1609  S   DMS A 101      19.762  39.489  18.350  1.00 25.99           S
+HETATM 1610  O   DMS A 101      19.279  37.904  18.777  1.00 23.69           Ox
+HETATM 1611  C1  DMS A 101      21.344  39.260  17.532  1.00 24.07           C
+HETATM 1612  C2  DMS A 101      18.750  40.066  17.029  1.00 20.50           C
+HETATM 1613  S   DMS A 102      22.157  39.211  12.217  1.00 27.26           S1
+HETATM 1614  O   DMS A 102      20.622  38.811  11.702  1.00 25.51           O
+HETATM 1615  C1  DMS A 102      22.715  40.764  11.522  1.00 26.00           C
+HETATM 1616  C2  DMS A 102      22.343  39.515  13.971  1.00 25.46           C
+TER    1617
+"""
+
```

Comment:
PEP8: I believe you need two empty lines here.

Reply:
I didn't really get this one. Do you mean something like this?

```
HETATM 1616  C2  DMS A 102      22.343  39.515  13.971  1.00 25.46           C
TER    1617
"""


def test_PDB_elements():
```

Reply:
Also, for @pytest.mark.filterwarnings, if I give it a string, there are several Internal errors and pytest exits with just 3/47 passed tests. It also doesn't point out the error which is weird but basically the module doesn't exist is a common Internal Error which has been popping

Reply:
Yeah so according to PEP8 there should be two empty lines between class or function definitions. According to flake8 this also extends to global variables like these PDB variables.

Note: you'll also want two lines above line 237.

Related note: You'll notice that this isn't consistent throughout `test_pdb`. As far as I understand this is because a lot of this code got written before PEP8 became the accepted standard for MDA. So our current approach is to ensure all future test is PEP8 compliant and fix PEP8 issues when we encounter them 😄. In case you don't already, if you use an IDE of some kind for your code (e.g. VScode, atom), I'd recommend you have flake8 installed so you can pick up these issues.

Reply:
> Also, for @pytest.mark.filterwarnings, if I give it a string, there are several Internal errors and pytest exits with just 3/47 passed tests. It also doesn't point out the error which is weird but basically the module doesn't exist is a common Internal Error which has been popping.
> Update: I don't think I need to have the UserWarning filter option. Since, we have made it more conservative, I think it is not popping any user valid for Invalid elements as all the PDBs used are valid.

I'm not sure I understand what's going on, do you want to push the code and we can have a look at it?

Reply:
Oh gotcha. My understanding was that there must be 2 blank lines between function definitions. I didn't know that we would need it for the string declaration and the function as well. 
I created a MDAnalysis workspace in XCode, I will check if there is a possible alternative to flake8 or maybe flake8 to install in there. This is helpful. 
Just pushing through the changes in a moment